Someday, when Solaris has GNU getopts, we won't need this kludge.  This allows
test/refresh-z to pass where it wants to use -z with an argument, but without
a space between the -z and the arg.


Index: test/refresh-z.test
===================================================================
--- test/refresh-z.test.orig	2008-11-05 14:10:52.524371000 -0600
+++ test/refresh-z.test	2008-11-05 14:20:59.946848000 -0600
@@ -17,7 +17,7 @@
 	> Fork of patch patches/test.diff created as patches/test-2.diff
 
 	$ echo baz >> foo
-	$ quilt refresh -zbaz.diff
+	$ quilt refresh -z baz.diff
 	> Fork of patch patches/test-2.diff created as patches/baz.diff
 
 	$ quilt series
